### PR TITLE
Add WebTorrent Desktop v0.3.1

### PR DESCRIPTION
**Pull request by Jerzerak**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [ ] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/webtorrent-desktop.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+cask 'webtorrent-desktop' do
```
Comment (vitorgalvao) [inline Casks/webtorrent-desktop.rb]: You haven’t followed the token reference. This needs to be `webtorrent`.

**Comment by Jerzerak** [comment]
Really? The name of the application is WebTorrent Desktop but it creates an app called WebTorrent, should I still chance it?

File: Casks/webtorrent-desktop.rb (hunk 1 above)
Comment (Jerzerak) [inline Casks/webtorrent-desktop.rb]: Really? The name of the application is WebTorrent Desktop but it creates an app called WebTorrent, should I still change it?

Comment (vitorgalvao) [inline Casks/webtorrent-desktop.rb]: Yes, the token reference is very strict. That’s the whole point. What each person interprets as the app’s name is subjective, and that leads to a lot of duplicates. It was a big issue in the past.

**Comment by Jerzerak** [comment]
Opening a new PR because I'm an idiot

**Pull request closed without merging**